protocol/daemons/liquidation/client/grpc_helper.go

```diff
@@ -9,16 +9,41 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/query"
 	"github.com/dydxprotocol/v4-chain/protocol/daemons/liquidation/api"
 	"github.com/dydxprotocol/v4-chain/protocol/lib/metrics"
+	blocktimetypes "github.com/dydxprotocol/v4-chain/protocol/x/blocktime/types"
 	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
 	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 )
 
+// GetPreviousBlockInfo queries a gRPC server using `QueryPreviousBlockInfoRequest`
+// and returns the previous block height.
+func GetPreviousBlockInfo(
+	ctx context.Context,
+	daemon *Client,
```

Comment:
meganit: can we rename this to be LiquidationDaemonClient? might mix it up with the queryClients